src/components/general/header/Header.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import Image from "next/image";
+import { NextLink } from "@mantine/next";
+import { Text, Burger, Button } from "@mantine/core";
+
+import styles from "./Header.module.css";
+
+import { staticData } from "../../../utils/staticData";
+const { header: COMPONENT_DATA } = staticData.components;
+
+const HeaderAction = () => {
+  const [opened, setOpened] = React.useState(false);
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.leftSection}>
+        <Burger
+          opened={opened}
+          onClick={() => setOpened(!opened)}
+          className={styles.burger}
+          size="sm"
+          color="white"
+          onBlur={() => setOpened(false)}
+        />
+        <div className={styles.logo}>
+          <Image
+            src={COMPONENT_DATA.logo.src}
+            alt={COMPONENT_DATA.logo.alt}
+            layout="intrinsic"
+          />
+        </div>
+      </div>
+      <div className={styles.links + (opened ? " " + styles.visible : "")}>
```

Comment:
nit: Use template literals

Reply:
> Any sort of link hover will come later? Thanks for the Vercel environment!

Not any as of now. imo mostly nav links with hover have some dropdown or some menu kind of interface generally. I have not seen much links with some hovers in this type of design.
Let me know what you think.

Reply:
We can always add it later on. Just wanted to check.